metadata.yaml

```diff
@@ -32,3 +32,6 @@ releaseSeries:
   - major: 1
     minor: 6
     contract: v1beta1
+  - major: 1
+    minor: 7
+    contract: v1beta1
```

Comment:
@cpanato It's already update here

Reply:
ah got it

you need to update the line 57 in test/e2e/config/gcp-ci.yaml

```
 - name: v1.6.99 # next; use manifest from source files
```

should be

```
 - name: v1.7.99 # next; use manifest from source files
```

Reply:
Looks like that fixed it 🙌 
Thank you 😊